src/Components/UserAccountPage/AccountDetails/Payments/CreditCard/CreditCard.css

```diff
@@ -0,0 +1,115 @@
+.atm-card-wrapper {
+  width: 20rem;
+  height: 12rem;
+  border-radius: 10px;
+  padding: 1rem;
+  overflow: hidden;
+  margin: 10px;
+  position: relative;
+}
+
+.atm-card-wrapper::-webkit-scrollbar {
+  display: none;
+}
+
+.atm-card-wrapper button {
+  visibility: hidden;
+  padding: 5px 0.8rem;
+  position: absolute;
+  z-index: 2;
+  top: 8px;
+  right: 6px;
+  border-radius: 50%;
+  background-color: var(--payment-card-hover);
+}
+
+.atm-card-wrapper button:hover {
+  background-color: var(--light-gray);
+}
+
+.atm-card-wrapper:hover button {
+  visibility: visible;
+}
+
+.card-option-menu {
+  right: 1rem;
+  top: 3rem;
+  width: 7rem;
+  height: auto;
+  border-radius: 4px;
+  color: var(--text-dark);
+  background-color: var(--primary-bg);
+  position: absolute;
+  z-index: 2;
```

Comment:
Suggestion: use variables instead of direct values in z-index for ease of maintenance of stacking context.

Reply:
noted